python/example_robot_data/robots_loader.py

```diff
@@ -281,11 +281,19 @@ class ICubReducedLoader(ICubLoader):
     urdf_filename = "icub_reduced.urdf"
 
 
+class Alexander(RobotLoader):
```

Comment:
To maintain code consistency 
```suggestion
class AlexanderLoader(RobotLoader):
```